torch_geometric/nn/models/graph_unet.py

```diff
@@ -0,0 +1,113 @@
+import torch
+import torch.nn.functional as F
+from torch_sparse import spspmm
+from torch_geometric.nn import TopKPooling, GCNConv
+from torch_geometric.utils import (add_self_loops, sort_edge_index,
+                                   remove_self_loops)
+from torch_geometric.utils.repeat import repeat
+
+
+class GraphUNet(torch.nn.Module):
+    r"""The Graph U-Net model from the `"Graph U-Nets"
+    <https://arxiv.org/abs/1905.05178>`_ paper which implements a U-Net graph
+    architecture with graph pooling and unpooling operations.
+
+    Args:
+        channels (int): Size of each sample.
+        depth (int): The depth of the U-Net architecture.
+        kernel_size (int or [int]): Size of the convolving kernel.
```

Comment:
Am I missing something ? This argument doesn't seem to be used

Reply:
Maybe due to a previous copy paste of docs ?

Reply:
Yes, thank you :)